plugins/channel_header_plug/channel_header_plug.jsx

```diff
@@ -122,6 +122,8 @@ export default class ChannelHeaderPlug extends React.PureComponent {
                 iconComponent={plug.icon}
                 onClick={() => plug.action(this.props.channel, this.props.channelMember)}
                 buttonId={plug.id}
+                tooltipKey={'plugin'}
+                tooltipText={plug.tooltipText}
```

Comment:
Would it make sense to default `tooltipText` to the plugin's `dropdownText` if it's not set?

Reply:
It sounds good. I've fixed 👍